lib/instance-validator.js

```diff
@@ -108,7 +108,7 @@ InstanceValidator.prototype._builtinValidators = function() {
   var validators = [];
   Utils._.forIn(this.modelInstance.rawAttributes, function(rawAttribute, field) {
     if (self.options.skip.indexOf(field) >= 0) {
-      return;
+      return false;
```

Comment:
Are you absolutely sure about that? lodash docs says you have to explicitely return false.
If it's currently exiting the loop early with just `return` that would be a bug.